### ISSUE TITLE
RFC: 

### ISSUE THREAD

**Issue by daniel-web-developer** [ossu/data-science#92]
The current links to MIT courses numbers 6.0001 and 6.0002 are to the courses on eDX, and you can only have them for free for a limited time. I suggest changing

I propose to change them to the courses on the MIT OCW website, which are, respectively, [6.0001](https://ocw.mit.edu/courses/6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016/) and [6.0002](https://ocw.mit.edu/courses/6-0002-introduction-to-computational-thinking-and-data-science-fall-2016/).

**Comment by waciumawanjohi** [comment]
That sounds reasonable. It looks like the OCW site provides the programming assignments, but no auto-grader. When I check the edx site it shows me a very old version of the course. Can you (or anyone watching this issue) report if auditing the edx course gives students access to an autograder?

**Comment by rootsmusic** [comment]
No @waciumawanjohi, "[audit track](https://support.edx.org/hc/en-us/articles/360013426573-What-are-the-differences-between-audit-free-and-verified-paid-courses-)" on edX doesn't grade anything.  It blocks assessments and exams.

**Comment by waciumawanjohi** [comment]
Thanks for the extra info.
Leaving this issue open for any additional comments.
@daniel-web-developer would you like to open a pull request tied to this issue (in the commit or the PR use the phrase "resolves #" and add this issue number (#92)? See [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more info.

**Comment by daniel-web-developer** [comment]
> Thanks for the extra info. Leaving this issue open for any additional comments. @daniel-web-developer would you like to open a pull request tied to this issue (in the commit or the PR use the phrase "resolves #" and add this issue number (#92)? See [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more info.

I think it's done.

**Comment by waciumawanjohi** [comment]
@daniel-web-developer yes, that PR looks good. Generally proposed changes to the curriculum are left open for 1 month to allow for relevant conversation. So I'll merge the PR in early August (assuming no one comments with a very good reason not to do so).

I'll be interested to see if the "Solves #92" works. I generally include it in the body of the PR/commit, while here it is in the title of the PR. We'll see!

**Comment by bradleygrant** [comment]
Anything we do here, we'll need to do likewise at https://github.com/ossu/computer-science/ since MIT 6.00.1x is recommended there (typically, we recommend students take either 6.00.1x via audit if near the start of an EdX run of the course, otherwise take 6.000000000000000001 via MIT OCW)

**Comment by bradleygrant** [comment]
Georgia Tech's CS1301 is a possibility:

**Comment by waciumawanjohi** [comment]
It looks like the 'closes #' does need to be in the  body of the PR, not the title.

Issue resolved.
